Xamarin.Forms.Platform.MacOS/Platform.cs

```diff
@@ -43,7 +43,16 @@ internal Platform()
 				var alert = NSAlert.WithMessage(arguments.Title, arguments.Cancel, arguments.Destruction, null, "");
 				if (arguments.Buttons != null)
 				{
-					alert.AccessoryView = GetExtraButton(arguments);
+					NSView extraButtons = GetExtraButton(arguments);
+					if (extraButtons.Frame.Height > 400) {
```

Comment:
we need something better instead of a magic 400 ..

Reply:
I agree that we need something better than a magic number.
So I'll go with 60% of the mac desktop screen height.
Is there something in Platform.cs that already knows the size of the mac desktop?

Reply:
Is this OK?
'''
int maxScrollHeight = (int)(0.6 * NSScreen.MainScreen.Frame.Height);
'''